doc/ref/states/aggregate.rst

```diff
@@ -122,7 +122,7 @@ This example, simplified from the pkg state, shows how to create mod_aggregate f
         for chunk in chunks:
             # The state runtime uses "tags" to track completed jobs, it may
             # look familiar with the _|-
-            tag = salt.utils.gen_state_tag(chunk)
+            tag = salt.utils.state.gen_tag(chunk)
```

Comment:
Since this would be in a state module, I think this can also use `__utils__`.

Reply:
👍  Fixed